snap/snapcraft.yaml

```diff
@@ -0,0 +1,35 @@
+#  Copyright (c) 2023, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+
+name: ispc
+base: core22 # ubuntu 22.04
+version: '1.19.0'
+license: BSD-3-Clause
+summary: Compiler for high-performance SIMD programming on the CPU
```

Comment:
Why n CPU only? Should we drop "on the CPU" part?

Reply:
fixed